specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/job.json

```diff
@@ -0,0 +1,931 @@
+                    "type": "number",
+                    "format": "double",
+                    "x-nullable": false,
+                    "description": "The progress made on the cleanup."
+                },
+                "failureInfo": {
+                    "type": "string",
+                    "description": "The information about any failures that have occurred."
+                },
+                "finishTime": {
+                    "type": "integer",
+                    "format": "int64",
+                    "x-nullable": false,
+                    "description": "The time at which the job completed."
+                },
+                "historyFile": {
+                    "type": "string",
+                    "description": "The history file of the job."
+                },
+                "jobACLs": {
+                    "$ref": "#/definitions/JobACLs",
+                    "description": "The ACLs of the job."
+                },
+                "jobComplete": {
+                    "type": "boolean",
+                    "x-nullable": false,
+                    "description": "Whether or not the job has completed."
+                },
+                "jobFile": {
+                    "type": "string",
+                    "description": "The job configuration file."
+                },
+                "jobId": {
+                    "type": "string",
+                    "description": "The full ID of the job."
+                },
+                "jobID": {
+                    "$ref": "#/definitions/JobID",
+                    "description": "The ID of the job."
+                },
+                "jobName": {
+                    "type": "string",
+                    "description": "The user-specified job name."
+                },
+                "jobPriority": {
+                    "type": "string",
+                    "description": "The priority of the job."
+                },
+                "mapProgress": {
+                    "type": "number",
+                    "format": "double",
+                    "x-nullable": false,
+                    "description": "The progress made on the maps."
+                },
+                "neededMem": {
+                    "type": "integer",
+                    "format": "int64",
+                    "x-nullable": false,
+                    "description": "The amount of memory needed for the job."
+                },
+                "numReservedSlots": {
+                    "type": "integer",
+                    "format": "int32",
+                    "x-nullable": false,
+                    "description": "The number of slots reserved."
+                },
+                "numUsedSlots": {
+                    "type": "integer",
+                    "format": "int32",
+                    "x-nullable": false,
+                    "description": "The number of slots used for the job."
+                },
+                "priority": {

```

Comment:
What is the difference between jobPriority and priority?

Reply:
They seem like same thing. However, the server side returns both. See [HDI_Job_Get.json](https://github.com/Azure/azure-rest-api-specs/pull/4489/files#diff-7064f25796a6df2c049307d5e1edbc07R18)

---
In reply to: [237732337](https://github.com/Azure/azure-rest-api-specs/pull/4489#discussion_r237732337) [](ancestors = 237732337)